bot/exts/info/information.py

```diff
@@ -435,8 +435,8 @@ async def user_messages(self, user: MemberOrUser) -> tuple[bool | str, tuple[str
             if e.status == 404:
                 activity_output = "No activity"
         else:
-            activity_output.append(f"{user_activity['total_messages']:,}" or "No messages")
-            activity_output.append(f"{user_activity['activity_blocks']:,}" or "No activity")
+            activity_output.append(f"{user_activity['total_messages']:,}" or "No messages")  # noqa: SIM222
+            activity_output.append(f"{user_activity['activity_blocks']:,}" or "No activity")  # noqa: SIM222
```

Comment:
Isn't this wrong currently? Even if `user_activity['total_messages']` is 0, `f"{0:,}"` is `"0"` which is truthy so "No messages" can never display?

Reply:
Indeed.
Fixed in the force push